### DISCUSSION TITLE
字体加载确实难，移动和桌面统计数据不一致，还有静态网址的问题

### DISCUSSION THREAD

**Discussion by chen-chenglong** [tangly1024/NotionNext#470] [General]
您好，很喜欢这个项目，可以专注到写作本身，不用考虑服务器了，谢谢！
提几个小问题：
1，字体加载确实慢，都是文字显示出来了等一会才变成目标字体，能不能直接用系统字体？
2，页面顶部的访问次数，移动端和桌面端差距很大，移动端刚发布的文章就300多次查看，应该不对，桌面端数据小很多。
3,因为之前一直用Wordpress 形成地址伪静态这x.com/123.html 的样式，您的项目可以自定义slug，我改为x.com/123.html 测试可以访问，但是不能刷新，因此目前还只能用x.com/123的样式，不知道html能不能写入代码，这样一个网址更适合分享，传播和收录，不知我的理解对不对。
谢谢🙏

**Comment by tangly1024** [comment]
- 嗨，感谢您的留言与反馈！

1. 系统字体当然是支持的，最早这个项目用的是系统的字体，速度飞快；然而美观和效率是一个不太容易权衡的问题哈。至于加快字体的加载，我也在找寻方案，下个版本我将尝试将字体文件作为本地导入，也许会更快一些。 https://github.com/tangly1024/NotionNext/issues/491
2. 页面访问次数使用的busuanzi的插件 https://busuanzi.ibruce.info/ 这个问题我将在另一个issue中跟进 https://github.com/tangly1024/NotionNext/issues/488
3. 结尾加入.html实现伪静态页面，是否适合分享传播收录，似乎并没有一个定论。NotionNext是 基于服务端渲染的技术，所以用户访问到 的页面即是静态渲染过的页面，只不过我将缓存时间设置的很短所以看起来是动态网站的效果。好运 的是，要实现`123.html`->`123`这样的伪静态在代码里的实现并不复杂，我将在下一个版本中加入这个特性：  https://github.com/tangly1024/NotionNext/issues/490

**Reply by chen-chenglong** [reply]
十分感谢！
第三条是123>123.html 即在slug 为123的时候，文章网址自动为123.html ，这个确实更适合搜索引擎

**Reply by tangly1024** [reply]
ok

**Comment by chen-chenglong** [comment]
另外这几天新发现的问题，搜索文章标题，百度可以检索到这个标题，点击后直达博客首页，而不是这个标题的文章页，不知是不是这个原因。

**Reply by tangly1024** [reply]
我确实也遇到此问题，搜索文章标题，但被百度索引的并非文章详情页，而是一些包含了本关键词的页面，例如文章首页，分类首页等。

可惜问题的原因暂未确认，但似乎在用google的时候并没遇到此问题

**Reply by chen-chenglong** [reply]
可能地址没有html结尾。
或者可能框架的问题早期discus Q也这样，不知道后续开发SEO修复了没有
